.github/ISSUE_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@ This issue tracker is only for technical issues related to bitcoin-core.
 
 General bitcoin questions and/or support requests and are best directed to the [Bitcoin StackExchange](https://bitcoin.stackexchange.com).
 
+For reporting security issues, please use [security@bitcoincore.org](mailto:security@bitcoincore.org).
```

Comment:
I think https://bitcoincore.org/en/contact/ will always be the page for contact, even if there is a subsection with a list of gpg keys.